app/dashboard/templates/ens/ens_rate_limit.html

```diff
@@ -10,7 +10,7 @@ <h2><label>{% trans "Address: "%}{{ owner }}</label></h2>
   </div>
   <div>
     <br>
-    {% trans "Only 1 request is allowed every "%}{{ limit_reset_days }}{% trans " days. If you have lost access to this wallet,"%} 
+    {% trans "Only 1 request is allowed every "%}{{ limit_reset_days }}{% trans "days. If you have lost access to this wallet,"%} 
```

Comment:
I believe this space is needed here!

Reply:
i have edited file solely on the comments in weblate. i do not know coding. due to limit_reset_days a space is needed before the text, i guess. please excuse.

Reply:
reverted changes with commit cfc87a82e58657ef22a208c2a3e7f31c3e2f7e76